webpack/assets/javascripts/react_app/components/Editor/components/EditorOptions.js

```diff
@@ -2,7 +2,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { Button, FormControl } from 'patternfly-react';
+import { FormControl } from 'patternfly-react';
```

Comment:
Do we still need this though?

It's used for `upload file` functionality, but I guess we can use `FileUploadField` from PF instead.